spec/support/factory_girl.rb

```diff
@@ -2,7 +2,8 @@
 
 FactoryGirl.define do
   factory :user do
-    sequence(:email) {|n| "user#{n}@example.com" }
+    sequence(:email) {|n| "lafiel.abriel#{n}@abhnation.com" }
+    sequence(:username) { |n| 'Lafiel_Abriel#{n}'}
```

Comment:
Unused block argument - `n`. You can omit the argument if you don't care about it.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.